extensions/amp-next-page/validator-amp-next-page.protoascii

```diff
@@ -13,43 +13,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the license.
 
-tags: {  # amp-document-recommendations
+tags: {  # amp-next-page
   html_format: AMP
   tag_name: "SCRIPT"
   extension_spec: {
-    name: "amp-document-recommendations"
+    name: "amp-next-page"
     allowed_versions: "0.1"
     allowed_versions: "latest"
   }
 }
-tags: {  # amp-document-recommendations: json config
+tags: {  # amp-next-page: json config
   html_format: AMP
   tag_name: "SCRIPT"
-  spec_name: "amp-document-recommendations extension .json configuration"
-  requires_extension: "amp-document-recommendations"
-  mandatory_parent: "AMP-DOCUMENT-RECOMMENDATIONS"
+  spec_name: "amp-next-page extension .json configuration"
+  requires_extension: "amp-next-page"
+  mandatory_parent: "AMP-NEXT-PAGE"
   attrs: {
     name: "recommendations"
```

Comment:
should this become `pages` instead of `recommendations`?

Reply:
Yup, makes more sense. Done.